packages/remix/test/fixtures-legacy/01-remix-basics/package.json

```diff
@@ -23,6 +23,6 @@
     "typescript": "^4.6.4"
   },
   "engines": {
-    "node": ">=14"
+    "node": "18.x"
```

Comment:
These Remix v1 tests were failing on Node 20 because of how Remix polyfills APIs such as `URLSearchParams`. So Node 20 would barf upon receiving the polyfilled implementations and the endpoint would fail with 500.